front/src/actions/message.js

```diff
@@ -74,15 +90,16 @@ function createActions(store) {
       });
       const messageText = state.currentMessageTextInput;
       try {
+        const id = uuid.v4();
         const newMessage = {
           text: messageText,
-          created_at: new Date()
+          created_at: new Date(),
+          id
         };
-        const tempId = uuid.v4();
         // we first push the message
         const newState = update(state, {
           messages: {
-            $push: [Object.assign({}, newMessage, { tempId })]
+            $push: [Object.assign({}, newMessage)]
```

Comment:
Still, why do you do `Object.assign({}, newMessage)` ?

No need to create a new object here, as there is no longer a new attribute to add

Reply:
Fixed